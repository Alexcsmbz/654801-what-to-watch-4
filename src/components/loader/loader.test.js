import Loader from './loader.jsx';

describe(`Loader snapshot test`, () => {
  it(`Loader should render loader`, () => {
    const tree = renderer.create(
        <Loader isLoading={true} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
