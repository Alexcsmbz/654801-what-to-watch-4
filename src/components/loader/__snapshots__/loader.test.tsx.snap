// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot test Loader should render loader 1`] = `
<div
  className="sc-AxjAm snkQt"
/>
`;
